pyproject.toml

```diff
@@ -83,6 +83,7 @@ pillow = {version = "*", optional=true}
 # server
 flask = {version = "^3.0", optional=true}
 flask-cors = {version = "^4.0", optional=true}
+json-repair = "^0.30.2"
```

Comment:
Place this line in an appropriate section (core)

Reply:
Fixed.